examples/games/game_menu.rs

```diff
@@ -779,18 +779,16 @@ mod menu {
             });
     }
 
-    fn menu_action(
-        interaction_query: Query<
-            (&Interaction, &MenuButtonAction),
-            (Changed<Interaction>, With<Button>),
-        >,
+    fn menu_action_with_clicks(
```

Comment:
I would keep the original name.

Reply:
Oh whoops forgot to change my testing name to back to the old one